pkg/utils/etcdutil/etcdutil.go

```diff
@@ -36,6 +36,9 @@ import (
 )
 
 const (
+	// DefaultEtcdClientTimeout is the default timeout for etcd client.
+	DefaultEtcdClientTimeout = 3 * time.Second
```

Comment:
Do we need to expose it?

Reply:
I am not sure whether is used by other servers.

Reply:
Once it will be used, we can expose it then.

Reply:
done